src/app/shared/player/player.component.html

```diff
@@ -1,4 +1,4 @@
-<div class="background-img" [style.background-image]="artworkSafe"></div>
+<div class="background-img" [style.background-image]="feedArtworkSafe"></div>
```

Comment:
It does default to the regular artworkurl later in the renderer as far as I recall, though I could be misremembering. The intention here is that the background album art comes from the feed level album art while the thumbnail comes from the episode level album art if available.